vm/systemSmartContracts/governance.go

```diff
@@ -633,6 +648,52 @@ func (g *governanceContract) closeProposal(args *vmcommon.ContractCallInput) vmc
 	return vmcommon.Ok
 }
 
+func (g *governanceContract) getAccumulatedFees() *big.Int {
+	currentData := g.eei.GetStorage([]byte(accumulatedFeeKey))
+	return big.NewInt(0).SetBytes(currentData)
+}
+
+func (g *governanceContract) setAccumulatedFees(value *big.Int) {
+	g.eei.SetStorage([]byte(accumulatedFeeKey), value.Bytes())
+}
+
+func (g *governanceContract) addToAccumulatedFees(value *big.Int) {
+	currentValue := g.getAccumulatedFees()
+	currentValue.Add(currentValue, value)
+	g.setAccumulatedFees(currentValue)
+}
+
+func (g *governanceContract) claimAccumulatedFees(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
+	if args.CallValue.Cmp(zero) != 0 {
+		g.eei.AddReturnMessage("callValue expected to be 0")
+		return vmcommon.UserError
+	}
+	if len(args.Arguments) != 0 {
+		g.eei.AddReturnMessage("invalid number of arguments expected 0")
+		return vmcommon.UserError
+	}
+	if !bytes.Equal(args.CallerAddr, g.changeConfigAddress) {
```

Comment:
I think at this point we can update to have an owner/admin variable instead of `changeConfigAddress`

Reply:
I did not wanted to add owner address in the toml file.